strategy/live_exec.py

```
@@ -160,15 +204,638 @@ def quote(condition_id: str, price: float, size: float, live: bool) -> None:
+        try:
+            order_status = client.get_order(order_id)
+            size_matched = float(order_status.get("size_matched", 0) if isinstance(order_status, dict) else getattr(order_status, "size_matched", 0) or 0)
+            if size_matched > 0:
+                cumulative_fills += 1
+                cumulative_loss_usd += size_matched * 0.01
+                print(f"\n  [FILL DETECTED] Order {order_id[:10]} matched {size_matched:.0f} shares ($ {size_matched*0.01:.2f})!")
+                if cumulative_fills >= max_fills or cumulative_loss_usd >= max_probe_loss_usd:
+                    print(f"\n[ABORT] Maximum probe loss cap reached ({cumulative_fills} fills, ${cumulative_loss_usd:.2f} loss). Halting probe immediately.")
+                    break
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)